src/amp-story-player/amp-story-player-impl.js

```diff
@@ -49,6 +49,9 @@ const IframePosition = {
 /** @const @type {!Array<string>} */
 const SUPPORTED_CACHES = ['cdn.ampproject.org', 'www.bing-amp.com'];
 
+/** @const @type {!Array<string>} */
+const SANDBOX_WHITELIST = ['allow-top-navigation'];
```

Comment:
You should not use the word `WHITELIST` in this code-base. The right term is allow-list (it is a bit ambigious because the elements literally start with `allow-`). However, this doesn't appear to be an allow-list in the classic sense. More of a minimum list.